components/service/glean/src/test/java/mozilla/components/service/glean/GleanTest.kt

```diff
@@ -26,8 +30,7 @@ import org.junit.Rule
 import org.junit.Test
 import org.junit.runner.RunWith
 import org.mockito.ArgumentMatchers.anyString
-import org.mockito.Mockito.doThrow
-import org.mockito.Mockito.spy
+import org.mockito.Mockito.*
```

Comment:
I'm not sure importing `*` is done anywhere else in the lib, can you just import the modules you need?  It's probably okay since it's in a test but the lib source files get hit for this by detekt.

Reply:
Yeah, I didn't iron out this issue, as the PR was marked as WIP. Good catch though, I've fixed this.